docs/api/v3.rst

```diff
@@ -0,0 +1,487 @@
+For example, here is the `Pip project's version screen`_.
+
+.. _Pip project's version screen: https://readthedocs.org/projects/pip/versions/
+
+
+Version detail
+++++++++++++++
+
+.. http:get::  /api/v2/project/(string:project_slug)/version/(string:version_slug)/
+
+    Retrieve details of a single version.
+
+    .. sourcecode:: js
+
+        {
+            "slug": "stable",
+            "verbose_name": "stable",
+            "identifier": "3a6b3995c141c0888af6591a59240ba5db7d9914",
+            "built": true,
+            "active": true,
+            "type": "tag",
+            "downloads": {
+                "pdf": "//readthedocs.org/projects/pip/downloads/pdf/stable/",
+                "htmlzip": "//readthedocs.org/projects/pip/downloads/htmlzip/stable/",
+                "epub": "//readthedocs.org/projects/pip/downloads/epub/stable/"
+            },
+            "links": [
+                {
+                    "href": "/api/v3/project/pip/version/stable/builds/",
+                    "rel": "builds",
+                    "type": "GET"
+                },
+                {
+                    "href": "/api/v3/project/pip/",
+                    "rel": "project",
+                    "type": "GET"
+                }
+            ]
+        }
+
+    :>json string slug: The version slug.
+    :>json string verbose_name: The name of the version.
+    :>json string identifier: A version control identifier for this version (eg. the commit hash of the tag)
+    :>json string built: Whether this version has been built
+    :>json string active: Whether this version is still active
+    :>json string type: The type of this version (typically "tag" or "branch")
+    :>json array downloads: URLs to downloads of this version's documentation
+    :>json array links: Array with HEATEOAS_ links to retrieve related information
+
+    :statuscode 200: Success
+    :statuscode 404: There is no ``Version`` with this slug
+
+
+Version edit
+++++++++++++
+
+.. http:patch::  /api/v3/project/(string:slug)/version/(string:slug)/
+
+    Edit a version.
+
+    **Example request**:
+
+    .. sourcecode:: js
+
+        {
+            "active": true,
+            "privacy_level": "public",
+            "tags": [
+                "python",
+                "packaging"
+            ]
+        }
+
+    :statuscode 204: Edited sucessfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+
+Builds
+~~~~~~
+
+Builds are created by Read the Docs whenever a ``Project`` has its documentation built.
+Frequently this happens automatically via a web hook but can be triggered manually.
+
+Builds can be viewed in the build screen for a project.
+For example, here is `Pip's build screen`_.
+
+.. _Pip's build screen: https://readthedocs.org/projects/pip/builds/
+
+
+.. TODO:
+
+   for filtering by ``commit`` we need the build listing --the other cases are useless
+
+
+Build detail
+++++++++++++
+
+.. http:get::  /api/v3/build/(int:id)/
```

Comment:
This probably shoulnd't be allowed (since doesn't make to much sense as a whole) and instead we should make it like `/api/v3/project/pip/builds/?commit=a1b2c3` or `?latest=true` or `?running=true`.